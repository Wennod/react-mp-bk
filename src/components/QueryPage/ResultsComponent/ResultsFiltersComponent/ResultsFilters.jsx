import React from 'react';

const ResultsPane = (props) => {
    if (props.isData) {
        return (
            <div className="result-filter">
                <p className="result-filter__count">{props.isData}</p>
                <div className="result-filter__controls">
                    <p className="result-filter__controls-title">Sort by</p>
                    <input type="button" value="release date"/>
                    <input type="button" value="rating"/>
                </div>
            </div>
        )
    } else {
        return null;
    }
}

export default ResultsPane;