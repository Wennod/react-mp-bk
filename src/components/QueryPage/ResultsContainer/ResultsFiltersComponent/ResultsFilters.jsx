import React from 'react';

import styles from './ResultsFilter.style.scss';

const ResultsPane = (props) => {
    if (props.isData) {
        return (
            <div className="result-filter">
                <p className="result-filter__count">{props.isData} movies found</p>
                <div className="result-filter__controls">
                    <p className="result-filter__controls-title">Sort by</p>
                    <input onClick={() => props.sortData('release date')} type="button" value="release date"/>
                    <input onClick={() => props.sortData('rating')} type="button" value="rating"/>
                </div>
            </div>
        )
    } else {
        return (
            <div className="result-filter result-filter--nodata">
            </div>
        );
    }
}

export default ResultsPane;