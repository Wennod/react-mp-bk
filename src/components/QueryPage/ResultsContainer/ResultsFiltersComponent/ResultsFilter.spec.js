import React from 'react';
import ResultsFilters from './ResultsFilters';

describe('ResultsContainer snapshot', () => {
    it('should match snapshot1', () => {
        const tree = mount(<ResultsFilters />);
        expect(tree).toMatchSnapshot();
    });
});

describe('ResultsContainer snapshot', () => {
    it('should match snapshot2', () => {
        const tree = mount(<ResultsFilters isData={false} />);
        expect(tree).toMatchSnapshot();
    });
});