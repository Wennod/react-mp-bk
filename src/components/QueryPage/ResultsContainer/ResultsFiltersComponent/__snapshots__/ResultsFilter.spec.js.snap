// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsContainer snapshot should match snapshot1 1`] = `
<ResultsPane>
  <div
    className="result-filter result-filter--nodata"
  />
</ResultsPane>
`;

exports[`ResultsContainer snapshot should match snapshot2 1`] = `
<ResultsPane
  isData={false}
>
  <div
    className="result-filter result-filter--nodata"
  />
</ResultsPane>
`;
